### DISCUSSION TITLE
In sequence diagram, how force two message aligned at the same level

### DISCUSSION THREAD

**Discussion by shutaozhenzhen** [plantuml/plantuml#1699] [Q&A] [answered]
```plantuml
@startuml
main -> main: command1
slave1 -> slave1: command2
slave2 -> slave2: command2
slave3 -> slave3: command2
slave4 -> slave4: command2
@enduml
```
how to make all command2 in the same line?

**Comment by The-Lum** [comment, marked as answer]
Hi @shutaozhenzhen,

Here is a proposal, using [teoz](https://plantuml.com/teoz):
```puml
@startuml
!pragma teoz true
main -> main: command1
slave1 -> slave1: command2
& slave2 -> slave2: command2
& slave3 -> slave3: command2
& slave4 -> slave4: command2
@enduml
```
>[![](https://www.planttext.com/api/plantuml/svg/SoWkIImgAStDuL8iA4fCpqrKA4dDhr8eAIfDvSfDpCnJqBLJ0D5M2idvkRc9UIc6NCKvYMMfXY0920ikPSIb1X4oWaiQOKeQmoMDCILDu98cI98EgNafG2S20000)](https://www.planttext.com/?text=SoWkIImgAStDuL8iA4fCpqrKA4dDhr8eAIfDvSfDpCnJqBLJ0D5M2idvkRc9UIc6NCKvYMMfXY0920ikPSIb1X4oWaiQOKeQmoMDCILDu98cI98EgNafG2S20000)

See also:
- https://plantuml.com/teoz

Regards,
Th.

**Reply by shutaozhenzhen** [reply]
Thanks
